### ISSUE TITLE
Hide balances

### ISSUE THREAD

**Issue by kolmansoifer** [cryptoadvance/specter-desktop#615]
Any chance you can add, as a default (but toggle-able), the hiding of balances on the wallet screens?

**Comment by stepansnigirev** [comment]
Good idea. Do you think it should be in general settings or hidden by default with the eyeball icon next to it?

**Comment by kolmansoifer** [comment]
Not sure....  I'm inclined to go with "default at wallet creation and toggleable using the eyeball icon thereafter".
If people want to unhide it and leave it unhidden, that is their choice.
Make sense?

**Comment by kolmansoifer** [comment]
Hmmm, now i'm not so sure about "default at wallet creation".
Ppl might freak out if they don't see their balances when they launch a new wallet.....
Maybe the default is "unhidden" but the eyeball icon is readily accessible.....
